src/coreclr/System.Private.CoreLib/CompatibilitySuppressions.xml

```diff
@@ -1,6 +1,22 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
```

Comment:
Why are these new suppressions needed?

Reply:
Oh that was done automatically by the build script, so I don't know. We could try reverting it and generating it again if one or more of these are confirmed to be unnecessary

Reply:
You have more than one month old sync of the main branch. Try rebasing against the current main.

It is not expected that the build scripts do edits like this.